iaso/odk/validator.py

```diff
@@ -64,13 +63,23 @@ def is_repeat_group(question):
 
 def get_list_name_from_select(question):
     q_type = question.get("type")
```

Comment:
Are we confident enough that `q_type` will never be `None`? Otherwise the code below would fail.

I'd suggest using `q_type  = question.get("type", "")`.

Reply:
technically it has survived to `is_select_one(question)` which would have filtered out question without a type.